convert.ts

```diff
@@ -19,9 +23,8 @@ export default async (yamlFilePath) => {
     }
 
     return await fieldMap.files.reduce((out, file) => {
-
         // open, check, and parse .csv or .xlsx
-        let data = parseFile(file);
+        let data = parseFile("testData/"+file);
```

Comment:
should probably revert this. I believe it works adding "testData/" in front of the file name in the yaml fieldmap. LMK your thoughts